www/src/data/tags-docs.js

```diff
@@ -0,0 +1,98 @@
+/* 
+  this is a whitelist of tags that can be displayed 
+  on the blog, the first value being the tag name, 
+  and the second value a link to an existing doc on
+  the topic, adding a new entry to this Map makes it
+  a new possible tag for blog posts
+*/
+export const TAGS_AND_DOCS = new Map([
+  [`accessibility`, `/docs/making-your-site-accessible/`],
+  [`agencies`, `/docs/gatsby-for-agencies/`],
+  [`algolia`, `/docs/adding-search-with-algolia/`],
+  [`apis`, ``],
+  [`apps`, ``],
+  [`authentication`, `/docs/building-a-site-with-authentication/`],
+  [`aws`, `/docs/deploying-to-s3-cloudfront/#getting-started---aws-cli`],
+  [`azure`, ``],
+  [`blogs`, ``],
+  [`building-sites-faster`, ``],
+  [`case-studies`, ``],
+  [`cdn`, ``],
+  [`cli`, ``],
+  [
+    `client-side-routing`,
+    `/docs/building-apps-with-gatsby/#client-only-routes--user-authentication`,
+  ],
+  [`cms`, `/docs/headless-cms/`],
+  [`code-editor`, ``],
+  [`community`, `/contributing/community/`],
+  [`content-mesh`, ``],
+  [`contentful`, `/docs/sourcing-from-contentful/`],
+  [`contest`, ``],
+  [`core-concepts`, `/docs/gatsby-core-philosophy/`],
+  [`cutting-edge-experiences`, ``],
+  [`data-visualization`, ``],
+  [`deployment`, `/docs/deploying-and-hosting/`],
+  [`developer-experience`, ``],
+  [`documentation`, `/docs/`],
+  [`documentation-sites`, ``],
+  [`drupal`, `/docs/sourcing-from-drupal/`],
+  [`ecommerce`, ``],
+  [`enterprise`, `/docs/gatsby-in-the-enterprise/`],
+  [`gatsby-apis`, `/docs/api-specification/#api`],
```

Comment:
```suggestion
  [`gatsby-apis`, `/docs/api-reference/`],
```
Should we link to API reference instead of going to the middle of that page? Or just go to the broader API specification page?

Reply:
I think the broader page is a good call